scripts/mod_loader/modui/pilot_arrange.lua

```diff
@@ -65,6 +99,16 @@ local function createUi()
 		-- update the global if we have not entered the hangar
 		if updateImmediately then
 			PilotList = pilots
+		else
+			-- alert the user to restart the game
+			modApi:scheduleHook(50, function()
+				sdlext.showButtonDialog(
+					GetText("OpenGL_FrameTitle"),
+					GetText("PilotArrange_RestartWarning_Text"),
+					responseFn, nil, nil,
+					{ GetText("OpenGL_Button_Quit"), GetText("OpenGL_Button_Stay") }
```

Comment:
You probably shouldn't re-use the OpenGL dialog functionality here, as it's pretty specific in that it's a one-time thing that also prevents the whole mod loader from working unless the game is restarted.

Generally, other popups should follow the principle of being only informational, while also giving the option to prevent the popup from showing (eg. `modui/popup_script-error.lua`)